content/01.abstract.md

```diff
@@ -1,10 +1,14 @@
 ## Abstract {.page_break_before}
 
-Important tasks in biology and biomedicine such as predicting gene functions, gene-disease associations, and drug repurposing opportunities are often framed as network edge prediction.
-Degree strongly influences edge prediction, and degree biases and degree imbalance can lead to nonspecific predictions.
-We introduce a network permutation framework to quantify the effect of node degree on network-based methods and prediction tasks.
-As part of this we define an edge prior to quantify the probability that two nodes are connected based only on their degree.
-After demonstrating that this prior feature shows excellent discrimination and calibration performance for 20 different biomedical networks (16 bipartite, 3 undirected, 1 directed), we conclude that our prior feature represents a suitable baseline for network link prediction tasks, as performance exceeding the baseline is attributable to factors other than degree alone.
-Additionally, we propose methods to incorporate network permutation and the edge prior into other predictive methods.
-Our results highlight the importance of degree for link prediction and provide software to account for its effects when degree bias may be present.
-We have released a full implementation of our network permutation method and the edge prior as an open-source Python package on GitHub and the Python Package Index.
+Important tasks in biomedical discovery, such as repurposing drugs or predicting the function of genes and their association to disease, are often framed as network edge prediction problems.
+In most real-world networks, the number of edges per node (called "degree") varies drastically.
+Since degree strongly influences edge prediction, any imbalance or bias in the distribution of degrees leads to nonspecific or misleading predictions.
+We introduce a framework to quantify the effect of node degree on edge prediction.
+Using degree-preserving network permutations, we define an "edge prior" to quantify the probability that two nodes are connected based only on their degree.
+Applying the edge prior to 20 different biomedical networks (16 bipartite, 3 undirected, 1 directed),
+we demonstrate that node degree information alone is capable of remarkable performance, with AUROCs frequently exceeding 85%.
+The predictive performance of degree diminishes on testing networks that were generated using distinct methods and hence show little degree correlation with the training network.
```

Comment:
Does this mean that thinking of degree (and consequently xswap) is not actually valuable? I know what you are going for here, but I don't know if most readers will.

Reply:
My position on this is that the edge prior is important to assess whether degree generalizes to the test set. For most test sets, it probably will. I think the point is that for some tests sets / applications, degree will generalize less. I would be OKAY removing this from the abstract, at least until we have a more rigorous recommendation on how to approach such a situation. Given the word count, I think this is potentially the least essential point.